WooCommerce/WooCommerceTests/Extensions/IconsTests.swift

```diff
@@ -966,37 +967,38 @@ final class IconsTests: XCTestCase {
     }
 
     func test_posNewTransactionImage_is_not_nil() {
-        XCTAssertNotNil(UIImage.posNewTransactionImage)
+        XCTAssertNotNil(Image(PointOfSaleAssets.newTransactionImageName))
```

Comment:
🤔 Writing a test for every added image is easy to forget.

To test this logic automatically, we could make `enum PointOfSaleAssets: CaseIterable`, define image names as cases, and have a single test that iterates through all the cases and checks if every one of them results in a non-nil image. What do you think?

Reply:
Hmm... a little conflicted initially, but I like the idea.

The tests will obviously be a lot better, but I wanted to avoid having to write `.rawValue` when we use the asset name. This can be avoided a few ways though – I'll take a look.

I'll also separate the tests to a new file, since we will want to move them out if we ever make POS a separate app.

Reply:
It also may be an unpopular idea but we could also skip writing tests such as these. I don't remember a situation where an asset has been deleted by accident, I assume these tests defend from situations such as this.

Reply:
Also against mistyped names of the assets, yeah.

I know what you mean... but if we can make it a lightweight test, that we don't need to rememeber, as you suggest, I think it'll have value. We don't get any help from the compiler on this so a test has more value than usual.